git_ignore.qmd

```diff
@@ -0,0 +1,142 @@
+    - That said, any member of your team who changed those settings would have the option to commit the corresponding change to the .Rproj file and the next time you **<span style="color:blue">pull</span>** you would have their settings instead
+
+#### Option B: Ignore the .Rproj
+
+We feel there are some serious reasons for ignoring the .Rproj as well:
+
+1. Risk of Deletion
+
+    - Say you decide to commit your .Rproj, what happens if someone deletes it and **<span style="color:green">pushes</span>** that deletion? This can happen if team members clone their version of the repo with a different name resulting in two local .Rproj files where one is redundant from that team member's perspective
+    - If a .Rproj is committed and then subsequently deleted, everyone who **<span style="color:blue">pulls</span>** that change will delete their local project. This results in an instant error that can be difficult to recognize if you don't know beforehand about this risk. The only fix is for all affected users to re-clone the repository
+    - So, if you add the .Rproj to the `.gitignore` preemptively, you can avoid the risk of this happening at some later stage; you could also add it as "`*.Rproj`" so regardless of the name of the file it will be ignored if the file type is .Rproj
+
+2. Loss of Flexibility
+
+    - When you commit your .Rproj you are "locking in" the name of your project. If your preliminary data exploration and analyses bring new hypotheses and goals to light you may want to change your project name to better match this new framing. If you had previously committed your .Rproj, it can be a hassle to change its name and **<span style="color:green">push</span>** this change
+    - Alternatively, you may want to change your GitHub repository name after publication to share keywords with your final manuscript title or the journal name to make the repository more easily searchable by your readers. Technically you could change the GitHub repository name without changing the corresponding .Rproj filename but there will be some disonance there if your readers clone your code
+
+3. Preserve Team Members' Local Filing Decisions
+
+    - Most working group members are involved in a staggering variety of pursuits in addition to their role in a synthesis working group. People tend to have their own filing systems in their computers to help them quickly and easily navigate among their different projects
+    - If you **commit** the .Rproj, you force everyone who clones your repository to use your naming convention which can be a hurdle (albeit likely a minor one) as they juggle their responsibilities
+
+#### Decision
+
+Ultimately, this is up to you and your team to decide! Hopefully our outlined reasons for and against adding the .Rproj to the `.gitignore` help inform this decision and we are happy to discuss this more if you have follow up questions!
+
+## Exceptions to the `.gitignore`
```

Comment:
I am not sure about "exceptions" in this section title? Seems more like a use case / example to me.

Reply:
Totally fair! Ping me when you're done making edits and I'll tweak this subheading accordingly!